composer.json

```diff
@@ -37,17 +37,19 @@
         "source": "https://github.com/MyIntervals/emogrifier"
     },
     "require": {
-        "php": "~7.0.0 || ~7.1.0 || ~7.2.0 || ~7.3.0",
+        "php": "~7.1.0 || ~7.2.0 || ~7.3.0",
         "ext-dom": "*",
         "ext-libxml": "*",
         "symfony/css-selector": "^2.8 || ^3.0 || ^4.0"
     },
     "require-dev": {
         "friendsofphp/php-cs-fixer": "^2.15.3",
-        "squizlabs/php_codesniffer": "^3.5.0",
-        "slevomat/coding-standard": "^5.0.4",
```

Comment:
Agree alphabetical order is better than, um, random.

Reply:
I'll make a separate PR for the config change, unless someone else makes the change while I'm at the day job?

Reply:
Go for it! :coffee:

Reply:
see #788